data/zilliqa/assets/zil1w5hwupgc9rxyuyd742g2c9annwahugrx80fw9h/meta.json

```diff
@@ -2,9 +2,14 @@
   "name": "Gary",
   "symbol": "GARY",
   "web": "http://thegarytoken.com",
-  "supply": "100",
+  "supply": 100,
+  "decimals": 4,
+  "email": "thegarytoken@gmail.com",
+  "publicTeam": true,
+  "links.github": "https://github.com/darthgus",
```

Comment:
that's not how links are provided.

Reply:
also this github doesnt contain any project related repo, remove

Reply:
done